.golangci.yml

```diff
@@ -11,7 +11,6 @@ linters:
     - gosimple
     - ineffassign
     - misspell
-    - staticcheck
```

Comment:
Temporary for now to fix out of memory issue. We can consider adding a separate TravisCI matrix job, running staticcheck by itself, or waiting for `staticcheck` 2019.1.1 support in `golangci-lint`:
* https://github.com/golangci/golangci-lint/pull/445
* https://github.com/dominikh/go-tools/releases/tag/2019.1.1

Reply:
Here's the memory usage difference locally for me:

```
# golangci-lint on master
6227824640  maximum resident set size
# golangci-lint 1.15.0
5142007808  maximum resident set size
```

We should also watch: https://github.com/dominikh/go-tools/issues/419

Reply:
Hi! https://github.com/golangci/golangci-lint/pull/445 was merged. Also, I've added [the section](https://github.com/golangci/golangci-lint#memory-usage-of-golangci-lint) about memory usage into README. You can try to set `GOGC=30` for example to trade memory for CPU.

Reply:
Thanks so much @jirfag 😄 This should be a huge help for us.